lib/collections/addon/provider/moderation/all/template.hbs

```diff
@@ -16,6 +16,16 @@
             </a>
         {{/let}}
     {{/each}}
+    <div data-test-submission-sort local-class='sort-wrapper'>
+        <PowerSelect
```

Comment:
@futa-ikeda I would ask for an aria-label here.

Reply:
Unfortunately, adding an `aria-label` to a PowerSelect won't add it onto the dropdown (at least not in this version of ember-power-select). Newer versions of ember-power-select have better accessibility, so when we upgrade we should do an audit and add them as needed